src/Json.Converter.fs

```diff
@@ -251,8 +251,8 @@ module Convert =
                         failwithf "Union of records of type '%s' cannot be deserialized with the value of the discriminator key is not a string to match against a specific union case" unionType.Name
             | otherwise ->
                 // TODO!!! Better error messages here
-                let unexpectedJson = JS.JSON.stringify otherwise
-                let expectedType = JS.JSON.stringify cases
+                let unexpectedJson = string otherwise
+                let expectedType = string cases
```

Comment:
We're removing inheritance of union types because of performance reasons so we're giving up some utility methods like `toJSON` to avoid repetition in every class definition. We can consider to add them again, but using the string operators works here anyways.